process/block/shardblock.go

```diff
@@ -868,6 +955,8 @@ func (sp *shardProcessor) receivedMetaBlock(metaBlockHash []byte) {
 			requestedBlockHeaders := sp.requestFinalMissingHeaders()
 			if requestedBlockHeaders == 0 {
 				areFinalAttestingHdrsInCache = true
+			} else {
+				log.Info(fmt.Sprintf("requested %d missing final meta headers\n", requestedBlockHeaders))
```

Comment:
we really have to think what we put into log.Info and log.Debug

Reply:
Done

Reply:
This info is also useful to follow if the requesting final headers mechanism is properly working how it has been designed to. We can do a refactor of all the log.Info when all these things will be stable.